sale_order_color_code_tintometric_system/models/account_move_line.py

```diff
@@ -0,0 +1,14 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    color_code_tintometric = fields.Char(compute="_compute_color_code_tintometric", string="Lot/S.Tin")
+
+    @api.depends('sale_line_ids.color_code_tintometric')
+    def _compute_color_code_tintometric(self):
+        for record in self:
+            record.color_code_tintometric = record.sale_line_ids.color_code_tintometric
```

Comment:
This shouldn't work if an invoice line is linked to more than one sale line. Try this (if there are more than one sale lines, we concatenate obtained values), if you don't urderstand the python expression, please ask:
```suggestion
            record.color_code_tintometric = ", ".join(record.sale_line_ids.filtered(lambda x: x.color_code_tintometric).mapped("color_code_tintometric"))
```